pkg/drawio/iconTreeNode.go

```diff
@@ -260,7 +260,7 @@ func newLoadBalancerTreeNode(parent SquareTreeNodeInterface, name string, privat
 
 type PrivateIPTreeNode struct {
 	abstractIconTreeNode
-	original bool // does the private IP was originally at the config file, or it is a fake one
+	original bool // does the private IP was originally at the config file, or is it just potential one
```

Comment:
is it an potential address or the entire relevant cidr? in the reports it seems the latter

Reply:
there is no change in the implementation, when creating a fake IP, we allocate for it one address, we just made an adjustment in the report - instead of printing the allocated address, we print the block